test/utils/http_static_test_server.py

```diff
@@ -28,6 +29,7 @@ class HTTPStaticTestServer(object):
     """
 
     def __init__(self, port=8088):
+        HTTPServer.address_family = socket.AF_INET6
```

Comment:
...as mentioned in the docstring that I can't actually find anywhere in the python docs:

https://github.com/python/cpython/blob/master/Lib/socketserver.py#L411-L418